scripts/publish-circleci.js

```diff
@@ -28,19 +28,18 @@ const {
  *
  * To run a prevalidation for the environment variables run `node scripts/publish-circleci.js -- -v`.
  */
+const nextVersion = process.env['SALESFORCEDX_VSCODE_VERSION'];
 
 if (process.argv.indexOf('-v') > -1) {
   console.log('Running prevalidation of environment variables.');
-  console.log(
-    `SALESFORCEDX_VSCODE_VERSION: ${process.env['SALESFORCEDX_VSCODE_VERSION']}`
-  );
+  console.log(`SALESFORCEDX_VSCODE_VERSION: ${nextVersion}`);
   console.log(`CIRCLECI_TOKEN: ${process.env['CIRCLECI_TOKEN']}`);
   console.log(`CIRCLECI_BUILD: ${process.env['CIRCLECI_BUILD']}`);
   process.exit();
 }
 
 checkEnvironmentVariables();
-checkBaseBranch(`release/v${process.env['SALESFORCEDX_VSCODE_VERSION']}`);
```

Comment:
For my own knowledge - What issue was being caused by this? From testing, I did not see anything in my local environment.

Reply:
I was running into a variable undefined error when running the git tag part
```
const gitTagName = `v${nextVersion}`;
                       ^

ReferenceError: nextVersion is not defined
    at Object.<anonymous> (/Users/lcamposguajardo/github/DevTools/salesforcedx-vscode/scripts/publish-circleci.js:75:24)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:837:10)
    at internal/main/run_main_module.js:17:11
```

Reply:
Ah, I gotcha. Thanks, that explains it